File: app/src/main/java/com/example/wpossbank/Modelos/SharedPreference.java
package com.example.wpossbank.Modelos;

import android.content.Context;
import android.content.SharedPreferences;

public class SharedPreference {
    SharedPreferences sharedPreferences;
    SharedPreferences.Editor editor;
    Context context;

    public SharedPreference(Context context) {
        this.context = context;
        sharedPreferences = context.getSharedPreferences("bd_shared", Context.MODE_PRIVATE);
        editor = sharedPreferences.edit();
    }

    public String getActiveUser(){
        return sharedPreferences.getString("usuario", "no encontrado");
    }

    public void setActiveUser(String user){
        editor.putString("usuario", user);
        editor.apply();
    }
}

File: app/src/main/java/com/example/wpossbank/models/SharedPreference.kt
package com.example.wpossbank.models

import android.content.SharedPreferences
import android.content.Context

class SharedPreference(var context: Context) {

    var sharedPreferences: SharedPreferences = context.getSharedPreferences("bd_shared", Context.MODE_PRIVATE)
    private var editor: SharedPreferences.Editor = sharedPreferences.edit()

    var activeUser: String?
        get() = sharedPreferences.getString("usuario", "no encontrado")
        set(userObjectId) {
            editor.putString("usuario", userObjectId)
            editor.apply()
        }

}